### PR TITLE
Milestone 2: Create - Remove book

### PR DESCRIPTION
**Pull request by Kingjosh007**
In this milestone, I followed [these instructions](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_create_and_remove_book.md) in order to implement the Add and Delete features.

### CODE REVIEW

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @Kingjosh007, 

You have done a good job so far 🚀  

## Highlights 🎉 

-  Imports create store, combineReducers, and applyMiddleware from redux.
-  Use combineReducers to create a reducer.
-  Defines ADD_BOOK and REMOVE_BOOK as action types
- Sets an initial state with an empty array

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

File: src/components/Books.js [hunk 1]
```diff
@@ -1,19 +1,44 @@
 /* eslint-disable react/prop-types */
```
Comment (bimbolabuari) [inline src/components/Books.js]: Let's remove all disabled Linters, Linting helps in cleaner and professional code ✊ 

Kindly check for other occurrences 🙂

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @Kingjosh007, 

You have done a good job so far 🚀  

## Highlights 🎉 

-  Imports create store, combineReducers, and applyMiddleware from redux.
-  Use combineReducers to create a reducer.
-  Defines ADD_BOOK and REMOVE_BOOK as action types
- Sets an initial state with an empty array

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @Kingjosh007  🙋 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request ❌ 

You have submitted a project, that has failing linter checks

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Kingjosh007** [comment]
Hi @AshakaE and @bimbolabuari 
From the image below, you can see that this Stylelint error doesn't come from my side, but from Github's. It seems the packages they need in order to perform the checks can't be installed at the moment, due to `registry.npmjs.org`'s unavailability. 
On my side everything is fine, so I just submitted without waiting for the check to complete. Should it really count as a wasted chance?

Microverse - Stylelint error not on my side, but on Github's

**Comment by Kingjosh007** [comment]
@AshakaE @bimbolabuari 
Still not working. I guess I'll just have to wait for `registry.npmjs.org` to be available again before submitting again.

**Comment by AshakaE** [comment]
> Hi @AshakaE and @bimbolabuari From the image below, you can see that this Stylelint error doesn't come from my side, but from Github's. It seems the packages they need in order to perform the checks can't be installed at the moment, due to `registry.npmjs.org`'s unavailability. On my side everything is fine, so I just submitted without waiting for the check to complete. Should it really count as a wasted chance?
> 
> Microverse - Stylelint error not on my side, but on Github&apos;s

**_Invalid Code Review Request does not count into the code reviews limit._**

**Comment by Davidosky007** [comment]
## Hi @Kingjosh007 👋, 

## Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
job-welldone-gif



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**